source/manuals/working-with-aws-accounts.html.md.erb

```diff
@@ -0,0 +1,344 @@
+---
+title: Working with AWS accounts
+last_reviewed_on: 2023-11-24
+review_in: 12 months
+---
+
+# <%= current_page.data.title %>
+
+Most teams in GDS use [Amazon Web Services (AWS)](https://aws.amazon.com/) as their infrastructure provider. GDS teams manage their own AWS accounts, but users must first sign into a shared base AWS account called `gds-users`. They can then assume roles in their team's AWS account to perform administrative tasks using [AWS's cross-account access pattern](https://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html).
```

Comment:
This is not how it works in DI, so I think we should clarify which teams it *does* apply to.

It'd be enough to have a note at the top saying 'this doesn't apply to DI', but I do think it's important to show (somehow) that this whole page is more of a manual for new users page, than guidance for how teams _should_ operate.

Reply:
Thanks, I have added @huwd 's suggested change to make it clear this guidance does not apply to One Login.

It's also worth noting that the guidance lives in the `manuals/` section of the GDS Way to reflect the fact that it is a howto rather than a policy.